packages/start-static/index.js

```diff
@@ -35,13 +35,14 @@ export default function () {
       proc.stderr.pipe(process.stderr);
     },
     async build(config) {
+      const appRoot = config.solidOptions.appRoot;
```

Comment:
Or something similar. We tend to avoid null coalescing and optional chaining for environment support.

Reply:
It's probably fine. I haven't actually looked at this yet. I've just got a radar for `??` and `?.` because I find myself literally going through every project removing them.